apps/src/javalab/Javalab.js

```diff
@@ -106,6 +110,14 @@ Javalab.prototype.init = function(config) {
   const onCommitCode = this.onCommitCode.bind(this);
   const onInputMessage = this.onInputMessage.bind(this);
   const onJavabuilderMessage = this.onJavabuilderMessage.bind(this);
+  const addPlaygroundItem = (itemId, itemData) =>
+    getStore().dispatch(addPlaygroundItemData(itemId, itemData));
+  const removePlaygroundItem = itemId =>
+    getStore().dispatch(removePlaygroundItemData(itemId));
+  const changePlaygroundItem = (itemId, itemData) =>
+    getStore().dispatch(changePlaygroundItemData(itemId, itemData));
+  const setPlaygroundItems = itemData =>
+    getStore().dispatch(setPlaygroundItemData(itemData));
```

Comment:
These items seem pretty specific to Playground. Can we import and set them up in Playground.js instead?